source/administration/changelog.md

```diff
@@ -168,12 +168,23 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 ### Compatibility  
 
-Backwards compatibility with the old CLI tool will be removed in v3.8. See [documentation to learn more about the new CLI tool](https://docs.mattermost.com/administration/command-line-tools.html).
+#### Removed and deprecated features
 
-Changes from v3.6 to v3.7:
+ - Removed `ServiceSettings: "SegmentDeveloperKey"` setting in `config.json`
+ - Backwards compatibility with the old CLI tool will be removed in April/2017 release. See [documentation to learn more about the new CLI tool](https://docs.mattermost.com/administration/command-line-tools.html).
+ - Deprecated APIv3 routes to be removed in April/2017 release:
```

Comment:
Will it be more clear for people to understand when the features are removed if we use the version number instead of the date? Or maybe use both?